server/files/stylesheets/docs.css

```
@@ -38,7 +38,9 @@ p:last-child {
 footer.footer {
   padding: 4em 1em !important;
 }
-
+.footer .ui.link.list > .item:hover svg g{
+  fill-opacity:1;
+}
```

As stated below i suggest to use the original x brand SVG and style it using CSS as follows.
The CSS will also take care about the same transition and color behavior as for the other list items.

```suggestion
.footer .ui.link.list > .item svg {
    width: 0.7em;
    height: auto;
    margin-left: 0.25em;
    margin-right: 0.25em;
    fill: rgba(0,0,0,.4);
    transition: 0.1s fill ease;
}
.footer .ui.link.list > .item:hover svg {
  fill: rgba(0,0,0,.8);
}
```